tools/sidekick/config.json

```diff
@@ -7,6 +7,13 @@
       "environments": ["edit"],
       "url": "/tools/sidekick/library.html",
       "includePaths": ["**.docx**"]
+    },
+    {
+      "id": "products-editor",
+      "title": "Edit Products",
+      "environments": ["edit"],
```

Comment:
I think we should also have 'dev' and 'preview' environments here.

Reply:
Many Thanks. I suppose the idea here is to just open the Products spread sheet directly via clicking "Edit Products" button in the ribbon. Once the spreadsheet is open - automatically via side kick, we should be able to see the "Preview" "Publish" the sheet after making our changes.